packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -57,249 +53,15 @@ class _DeepLinkPageState extends State<DeepLinkPage>
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              const Text('Selected Deep link validation details'),
-              IconButton(
-                onPressed: () =>
-                    controller.showSpitScreenNotifier.value = false,
-                icon: const Icon(Icons.close),
-              ),
-            ],
-          ),
-          Text(
-            'This tool assistants helps you diagnose Universal Links, App Links,'
-            ' and Custom Schemes in your app. Web check are done for the web association'
-            ' file on your website. App checks are done for the intent filters in'
-            ' the manifest and info.plist file, routing issues, URL format, etc.',
-            style: Theme.of(context).textTheme.bodySmall,
-          ),
-          const Text('Domain check'),
-          Expanded(child: _DomainCheckTable(linkData: linkData)),
-        ],
-      ),
-    );
-  }
-}
-
-class _DomainCheckTable extends StatelessWidget {
-  const _DomainCheckTable({
-    required this.linkData,
-  });
-
-  final LinkData linkData;
-
-  @override
-  Widget build(BuildContext context) {
-    return DataTable(
-      columns: const [
-        DataColumn(label: Text('OS')),
-        DataColumn(label: Text('Issue type')),
-        DataColumn(label: Text('Status')),
-      ],
-      rows: [
-        if (linkData.os.contains('Android'))
-          DataRow(
-            cells: [
-              const DataCell(Text('Android')),
-              const DataCell(Text('Digital assets link file')),
-              DataCell(
-                linkData.domainError
-                    ? Text(
-                        'Check failed',
-                        style: TextStyle(
-                          color: Theme.of(context).colorScheme.error,
-                        ),
-                      )
-                    : Text(
-                        'No issues found',
-                        style: TextStyle(
-                          color: Theme.of(context).colorScheme.green,
-                        ),
-                      ),
-              ),
-            ],
-          ),
-        if (linkData.os.contains('iOS'))
-          DataRow(
-            cells: [
-              const DataCell(Text('iOS')),
-              const DataCell(Text('Apple-App-Site-Association file')),
-              DataCell(
-                Text(
-                  'No issues found',
-                  style: TextStyle(color: Theme.of(context).colorScheme.green),
-                ),
-              ),
-            ],
-          ),
-      ],
+    return ValueListenableBuilder(
+      valueListenable: controller.selectedProject,
+      builder: (_, FlutterProject? project, __) {
+        if (project == null) {
+          return const SelectProjectView();
+        } else {
+          return const DeepLinkListView();
```

Comment:
nit: this can be condensed as a ternary 
`return project == null ? const SelectProjectView() : DeepLinkListView();`